src/core/api/checkout/checkout_ecommerce/v1/_openapi.json.tpl

```diff
@@ -111,6 +111,18 @@
     "/transactions": {
       "post": {
         "operationId": "newTransaction",
+        "parameters": [
```

Comment:
To delete, this is parameter of `pagopa-ecommerce-transaction-service` and not of `ecommerce-checkout` API. So, the client must not set this header.

Reply:
fixed in https://github.com/pagopa/pagopa-infra/pull/607/commits/6e372723ccde22af54cf8182cc3ed44bac0138ab